README.md

```diff
@@ -461,10 +461,19 @@ node followerDiff.js
 ## Optional environment variables
 
 * **LISTING_DIGESTS_INPUT_DIR**: read listing digests from the
-    specified LevelDB database
+  specified LevelDB database
 
 * **PARALLEL_SCANS**: number of databases to scan in parallel (default 4)
 
+* **EXCLUDE_FROM_CSEQS**: cseq values for a set of raft sessions, to
```

Comment:
can this sentence be rewritten in a clearer way? Maybe an example of what gets excluded could help too

Reply:
done, let me know if it's clearer (I pretty much copied what I added in https://github.com/scality/s3utils/pull/246)

Reply:
I can add an example as well indeed.